doc/lightningd-config.5.md

```diff
@@ -463,6 +463,25 @@ if C-lightning terminates.
 Built-in plugins, which are installed with lightningd(8), are automatically
 considered important.
 
+### Experimental Options
+
+Experimental options are subject to breakage between releases: they
+are made available for advanced users who want to test proposed
+features.  If lightningd is built with `--enable-experimental-features`
```

Comment:
Might not be totally obvious what "is built with" means from the outset; might be good to give people a clue via `configured`?

```
If lightningd is built configured with ...
```